KoreBuild-dotnet/build/install.cmd

```diff
@@ -1,10 +1,10 @@
 @Echo off
 
-for /f "delims=" %%i in ('PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.IO.Path]::GetTempFileName()"') do set DNVM_CMD_PATH_FILE=%%i.cmd
+for /f "delims=" %%i in ('PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.IO.Path]::GetTempFileName()"') do set INSTALL_CMD_PATH_FILE=%%i.cmd
```

Comment:
I lied. This is our old `dnvm.cmd` file: https://github.com/aspnet/Universe/blob/dev/build/dnvm.cmd it's been like this for ages